### PR TITLE
added styling to project

### PR DESCRIPTION
**Pull request by Mithi-code**
### Styling

- Added color according to requirement

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -22,12 +22,11 @@ function Form() {
...
-        <br />
-        <input type="text" placeholder="Please select categories" value={category} onChange={(e) => setCategory(e.target.value)} required />
-        <br />
-        <button type="submit">Submit</button>
+        <input className="title" type="text" placeholder="Please enter title" value={title} onChange={(e) => setTitle(e.target.value)} required />
+        <input className="category" type="text" placeholder="Please select categories" value={category} onChange={(e) => setCategory(e.target.value)} required />
+        <button type="submit" className="btn-submit">ADD BOOK</button>
       </form>
     </>
   );
```
Comment (Emmyn5600) [inline src/components/Form.js]: - [ ] Kindly, compare your implementation with the design provided to meet all the requirements.

Original design

header--2

Your design

header--4

File: src/components/Navbar.js [hunk 2]
```diff
@@ -3,8 +3,9 @@ import { Link } from 'react-router-dom';
...
 const Navbar = () => (
   <>
-    <nav>
-      <ul>
+    <nav className="head">
+      <h1>Bookstore CMS</h1>
+      <ul className="links">
         <li><Link to="/">Books</Link></li>
         <li><Link to="/categories">Categories</Link></li>
       </ul>
```
Comment (Emmyn5600) [inline src/components/Navbar.js]: - [ ] Kindly, compare your implementation with the design provided to meet all the requirements

Original design

header--1

Your design

header--3

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @Mithi-code 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

 ## Required Changes ♻️
- Check the comments under the review.

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Book.js [hunk 3]
```diff
@@ -15,14 +16,43 @@ function Book({
...
-        <button type="button" onClick={remove}>Remove</button>
-      </li>
-    </>
+    <div className="book-cards">
+      <div className="book-list">
+        <p className="book-category">{category}</p>
+        <p className="book-title">{title}</p>
+        <button type="button" className="btn-modify">Comment</button>
+        <button type="button" onClick={remove} className="btn-modify">Remove</button>
+        <button type="button" className="btn-modify">Edit</button>
```
Comment (elyor-doniyorov) [inline src/components/Book.js]: - [ ] In order to stick the design as much as possible as the previous reviewer requested, kindly put bars between `Comment Remove Edit` buttons and also make `category names` bolder below is the original design to compare👇
Screenshot (142)_LI

- Your design 👇 
Screenshot (145)

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Mithi-code, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @Mithi-code, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



 ### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**